languages/swift/config.json

```diff
@@ -5,7 +5,49 @@
   "version": 3,
   "online_editor": { "indent_style": "space", "indent_size": 2 },
   "exercises": {
-    "concept": [],
+    "concept": [
+      {
+        "slug": "arrays",
+        "uuid": "b6c532c9-1e89-4fbf-8f08-27f5befb5bb8",
+        "concepts": ["arrays"],
+        "prerequisites": ["basics"]
+      },
+      {
+        "slug": "basics",
+        "uuid": "E6143DBC-895E-47AA-B42C-3034E1F872C2",
```

Comment:
FYI we usually generate UUID's using the https://www.uuidgenerator.net/version4 website, which uses lowercase. But I don't think it matters, as it is still unique, right @iHiD?

Reply:
I'd rather they were lowercase, yes. But we'll probably do a big pass/update at the end before launch anyway.